### PR TITLE
OpenApi fixes

### PR DESCRIPTION
**Pull request by Rens-Braspenning**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

**Comment by dougelliott** [comment]
@Rens-Braspenning Checking in on things regarding the conflict resolution. Any questions we could help with?

File: certified-connectors/ShareEffect/apiDefinition.swagger.json [hunk 1]
```diff
@@ -50,8 +50,8 @@
                   "type": "string",
                   "x-ms-dynamic-values": {
                     "operationId": "GetTermGroups",
-                    "value-path": "key-item-output",
-                    "value-title": "key-item-output"
+                    "value-path": "",
+                    "value-title": ""
```
Comment (sriyen-msft) [inline certified-connectors/ShareEffect/apiDefinition.swagger.json:54]: @Rens-Braspenning Please remove the empty value properties i.e. "value-path" and "value-title"

File: certified-connectors/ShareEffect/apiDefinition.swagger.json [hunk 2]
```diff
@@ -66,8 +66,8 @@
                         "parameter": "termsgroup"
                       }
                     },
-                    "value-path": "key-item-output",
-                    "value-title": "key-item-output"
+                    "value-path": "",
+                    "value-title": ""
```
Comment (sriyen-msft) [inline certified-connectors/ShareEffect/apiDefinition.swagger.json:70]: @Rens-Braspenning Please remove the empty value properties i.e. "value-path" and "value-title"

File: certified-connectors/ShareEffect/apiDefinition.swagger.json [hunk 3]
```diff
@@ -76,8 +76,8 @@
                         "parameterReference": "body/termsgroup"
                       }
                     },
-                    "itemValuePath": "key-item-output",
-                    "itemTitlePath": "key-item-output"
+                    "itemValuePath": "",
+                    "itemTitlePath": ""
```
Comment (sriyen-msft) [inline certified-connectors/ShareEffect/apiDefinition.swagger.json:80]: @Rens-Braspenning Please remove the empty value properties i.e. "value-path" and "value-title"

File: certified-connectors/ShareEffect/apiDefinition.swagger.json [hunk 4]
```diff
@@ -145,8 +142,8 @@
                   "type": "string",
                   "x-ms-dynamic-values": {
                     "operationId": "GetTermGroups",
-                    "value-path": "key-item-output",
-                    "value-title": "key-item-output"
+                    "value-path": "",
+                    "value-title": ""
```
Comment (sriyen-msft) [inline certified-connectors/ShareEffect/apiDefinition.swagger.json:146]: @Rens-Braspenning Please remove the empty value properties i.e. "value-path" and "value-title"

File: certified-connectors/ShareEffect/apiDefinition.swagger.json [hunk 5]
```diff
@@ -161,8 +158,8 @@
                         "parameter": "termsgroup"
                       }
                     },
-                    "value-path": "key-item-output",
-                    "value-title": "key-item-output"
+                    "value-path": "",
+                    "value-title": ""
```
Comment (sriyen-msft) [inline certified-connectors/ShareEffect/apiDefinition.swagger.json:162]: @Rens-Braspenning Please remove the empty value properties i.e. "value-path" and "value-title"

File: certified-connectors/ShareEffect/apiDefinition.swagger.json [hunk 6]
```diff
@@ -171,8 +168,8 @@
                         "parameterReference": "body/termsgroup"
                       }
                     },
-                    "itemValuePath": "key-item-output",
-                    "itemTitlePath": "key-item-output"
+                    "itemValuePath": "",
+                    "itemTitlePath": ""
```
Comment (sriyen-msft) [inline certified-connectors/ShareEffect/apiDefinition.swagger.json:172]: @Rens-Braspenning Please remove the empty value properties i.e. "value-path" and "value-title"

**Pull request merged**